bleachbit/GuiPreferences.py

```diff
@@ -178,6 +178,11 @@ def __general_page(self):
             'toggled', self.__toggle_callback, 'delete_confirmation')
         vbox.pack_start(cb_popup, False)
 
+        # Use base 1000 oder 1024?
+        cb_units_iec = gtk.CheckButton(_("Use IEC sizes (1 KiB = 1024 bytes) instead of SI (1 kB = 1000 bytes)"))
```

Comment:
I also thought of using a ComboBox, but it's quite cumbersome without `gtk.combo_box_new_text()` (which is deprecated in PyGTK 2.24) and the replacement `gtk.ComboBoxText` was added in 2.24.
If it's ok for you, I'd have a go at it after the transition to GTK 3.